app/lib/geordi-logger.coffee

```diff
@@ -4,12 +4,21 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
   constructor: (@state, @geordi) ->
 
   @tokens = ['zooHome', 'zooTalk', 'zooniverse/gravity-spy']
+  @GEORDI_WARNINGS_TIMEOUT_INTERVAL = 5 * 60 * 1000 # only warn about Geordi down once every 5 mins, to avoid console spam
```

Comment:
Caps are fine here. Normally constants are defined as variables, not as class properties.